File: Iniciante/1010 - Cálculo Simples/Python/1010 - Cálculo Simples.py
# pega 3 valores na mesma linha e atribui a variáveis
cod1, num1, valor1 = input().split(' ')
cod2, num2, valor2 = input().split(' ')

# Converte o valor para os tipos necessários 
cod1 = int(cod1)
num1 = int(num1)
valor1 = float(valor1)

cod2 = int(cod2)
num2 = int(num2)
valor2 = float(valor2)

total = (num1*valor1) + (num2*valor2)

print('VALOR A PAGAR: R$ {:.2f}'.format(total))

File: Iniciante/1010 - Cálculo Simples/Java/1010 - Cálculo Simples.java
import java.util.Locale;
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		// objeto para leitura dos dados de entrada
		Scanner leitor = new Scanner(System.in).useLocale(Locale.US);

		// recebe os valores na mesma linha
		int cod1 = leitor.nextInt();
		int qtd1 = leitor.nextInt();
		double valorUnit1 = leitor.nextDouble();

		// recebe os valores na mesma linha
		int cod2 = leitor.nextInt();
		int qtd2 = leitor.nextInt();
		double valorUnit2 = leitor.nextDouble();

		// cálculo do total
		double total = (qtd1*valorUnit1) + (qtd2 * valorUnit2);
        
		// saída
		System.out.printf("VALOR A PAGAR: R$ %.2f\n", total);
		
	}

}